.github/workflows/on-pull_request-opened-synchronize-reopened.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           path: aerius-github-actions
           ref: v1
```

Comment:
Shouldn't this be v1.0.2 as well?

Reply:
Ah yes, apparently this was never updated to 1.0.2 like the rest in a previous PR.